packages/accordion/package.json

```diff
@@ -9,25 +9,26 @@
     "url": "git+https://github.com/reach/reach-ui.git",
     "directory": "packages/accordion"
   },
-  "scripts": {
-    "build": "ts-node --transpile-only ../../scripts/build-package $npm_package_name"
-  },
   "dependencies": {
     "@reach/auto-id": "0.13.0",
     "@reach/descendants": "0.13.0",
     "@reach/utils": "0.13.0",
+    "prop-types": "^15.7.2",
     "tslib": "^2.0.0"
   },
   "peerDependencies": {
     "react": "^16.8.0 || 17.x",
     "react-dom": "^16.8.0 || 17.x"
   },
-  "main": "dist/index.js",
-  "module": "dist/accordion.esm.js",
-  "typings": "dist/index.d.ts",
```

Comment:
maybe you would be willing to help with this issue: https://github.com/preconstruct/preconstruct/issues/259 ? :>